app/Website/routes/website-routes.php

```diff
@@ -15,3 +16,5 @@
 Route::get('/blog-post-tags/{tag}', [BlogPostTagsController::class, 'show'])->name('blog-post-tags.show');
 
 Route::post('/contact', ContactController::class)->name('contact')->middleware('throttle:5,1');
+
+Route::post('newsletter/subscribe', [NewsletterSubscribeController::class, 'subscribe']);
```

Comment:
Instead of `subscribe` I would use one of the 5 API RESTful actions - index, show, store, update, destroy - in this case `store` if you are saving a new subscriber.

Reply:
Changed and used the store restful action.